.github/workflows/pr-e2e-tests.yaml

```diff
@@ -71,4 +71,11 @@ jobs:
         run: |
           count=0; until curl -s --fail localhost:8080/v1/.well-known/ready; do ((count++)); [ $count -ge 10 ] && echo "Reached maximum retry limit" && exit 1; sleep 15; done
       - name: Run tests
-        run: poetry run pytest ./tests/e2e
+        shell: bash
+        run: |
+          if [[ "${{ matrix.neo4j-edition }}" == "community" ]]; then
+              poetry run pytest -m 'not enterprise_only' ./tests/e2e
```

Comment:
Why run with this pytest mark?

Reply:
This file is not supposed to be updated, let me check what happened (probably a wrong merge)

Reply:
Should be fixed after merge with main.